docs/getting_started_guide/Installation_Options/r10k/Installing_SIMP_using_r10k.rst

```diff
@@ -134,10 +137,10 @@ connect to your puppetserver as defined in ``simp_options`` and run puppet a few
 times in order to get the new system in order.
 
 .. WARNING::
-   SIMP, by default, implements ``tcpwrappers`` and PAM access restrictions.
-   The root user should always be able to log in at a console, but if there is no
-   console, like in `AWS`_, be sure to add a user to the PAM whitelist and give
-   it sudo powers:
+
+   SIMP, by default, implements :term:`tcpwrappers` and :term:`PAM` access
+   restrictions.  Be sure to add a user to the PAM whitelist and give it
+   ``sudo`` powers:
```

Comment:
Line 134 above:  runpuppet is deprecated.
Line 143: Should this mention potential lockout, if no such user with ssh and sudo privileges exists?